### PR TITLE
Add __init__ and apply layer normalization on text only.

### PR DESCRIPTION
**Pull request by boru-roylu**
1. Add `__init__.py` in the directory `modeling.model`.
2. Apply layer normalization on text only.
3. Fix Some small nits.

### CODE REVIEW

**Review by santiaontanon-google — APPROVED** [review]
Thanks Roy, LGTM!

**Review by wqfcr-google — APPROVED** [review]

File: src/modeling/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from modeling.models.mmt_encoder import MmtEncoder
+from modeling.models.mmt_pretraining_model import MmtPretrainingModel
+from modeling.models.mmt_classification_model import MmtClassificationModel
```
Comment (jtainslie) [inline src/modeling/models/__init__.py:3]: Imports should be for packages or modules, not individual classes.  (go/pystyle#imports)

File: src/modeling/models/mmt_pretraining_model.py [hunk 2]
```diff
@@ -26,33 +26,38 @@ class MmtPretrainingModel(tf.keras.Model):
...
   """
 
   def __init__(self,
                encoder: tf.keras.Model,
-               mpp_output_num_classes=None,
-               mlm_activation=None,
-               mlm_initializer='glorot_uniform',
-               mpp_activation=None,
-               mpp_initializer='glorot_uniform',
+               mpp_output_num_classes: int = None,
```
Comment (jtainslie) [inline src/modeling/models/mmt_pretraining_model.py]:
```suggestion
               mpp_output_num_classes: Optional[int] = None,
```

File: src/modeling/models/mmt_pretraining_model.py [hunk 3]
```diff
@@ -26,33 +26,38 @@ class MmtPretrainingModel(tf.keras.Model):
...
+               mlm_initializer: str = 'glorot_uniform',
+               mpp_activation: Optional[Union[Callable, str]] = None,
+               mpp_initializer: str ='glorot_uniform',
                classification_heads: Optional[List[tf.keras.layers.Layer]] = None,
+               bind_word_embedding_table: bool = True,
                name: str = 'mmt_pretraining_model',
                **kwargs):
+    """instantiates `MmtPretrainingModel`.
+
+    Args:
```
Comment (jtainslie) [inline src/modeling/models/mmt_pretraining_model.py:44]: Some arguments look missing in this docstring.

**Review by jtainslie — APPROVED** [review]
Readability lgtm.
